packages/units/src/getBoundRect.js

```diff
@@ -130,6 +131,16 @@ export function getCorners(angle, x, y, width, height) {
   };
 }
 
+export function isElementBelowLimit(element, verifyLink = true) {
```

Comment:
let's keep this in its separate `isElementBelowLimit.js` file (within units package).

This `getBoundRect.js` file is already a bit long, and I think we should rather split that up into 1 file per exported function

Reply:
I got a dependency look when I did this. Thoughts on that.

Reply:
Could you perhaps elaborate?

Reply:
Fixed.